alerts/openshift-virtualization-operator/HCOMisconfiguredDescheduler.md

```diff
@@ -0,0 +1,70 @@
+# HCOMisconfiguredDescheduler
+
+## Meaning
+
+A Descheduler is a OpenShift Container Platform application that causes the
+control plane to
+re-arrange the workloads in a better way.
+
+The descheduler uses the OpenShift Container Platform eviction API to evict
+pods, and receives
+feedback from `kube-api` whether the eviction request was granted or not.
+On the other side, in order to keep VM live and trigger live-migration,
+OpenShift Virtualization handles eviction requests in a custom way and
+unfortunately
+a live migration takes time.
+So from the descheduler's point of view, `virt-launcher` pods fail to be
+evicted, but they actually migrating to another node in background.
+So the way OpenShift Virtualization handles eviction requests causes the
+descheduler to
+make wrong decisions and take wrong actions that could destabilize the cluster.
```

Comment:
-> Therefore, when a `virt-launcher` pod is migrating to another node in the background, the descheduler detects this as a pod that failed to be evicted. As a consequence, the manner in which OpenShift Virtualization handles eviction requests causes the descheduler to make incorrect decisions and take incorrect actions that might destabilize the cluster.